pkgs/dart_mcp_server/lib/src/utils/cli_utils.dart

```diff
@@ -224,7 +225,10 @@ Future<CallToolResult> runCommandInRoot(
   final result = await processManager.run(
     commandWithPaths,
     workingDirectory: workingDir.path,
-    runInShell: true,
+    runInShell:
+        // Required when running .bat files on windows, but otherwise should
+        // be avoided.
```

Comment:
Maybe worth expanding on this comment a little so it's clear this is because of escaping?